src/Components/WebAssembly/testassets/StandaloneApp/StandaloneApp.csproj

```diff
@@ -9,4 +9,25 @@
     <Reference Include="System.Net.Http.Json" />
   </ItemGroup>
 
+  <Target Name="_FixupNpmProjItem" BeforeTargets="ResolveReferencedProjectsStaticWebAssets" AfterTargets="ResolveProjectReferences">
```

Comment:
Will this be removed later?

Reply:
Yep, if you see is in a "Workarounds commit"

Reply:
I picked up an updated SDK so this is undone